docs/source/using/run-against-an-existing-machine.adoc

```diff
@@ -84,9 +84,16 @@ Use the following command on the host to run {project} against a remote machine:
 
 ----
 $ minishift start --vm-driver generic --remote-ipaddress <remote_IP_address> --remote-ssh-user <username> --remote-ssh-key <private_ssh_key>
+$ minishift addon apply htpasswd-identity-provider
```

Comment:
@praveenkumar it should be `$ minishift addon apply htpasswd-identity-provider --addon-env  USER_PASSWORD=<NEW_PASSWORD>`

Reply:
@LalatenduMohanty by default if the user doesn't provide the password then it will take `developer` but I think it good to put that in document.

Reply:
+1, in this use case it is nice to show how to set non-default password

Reply:
+1. It would also be worth noting what the default is if no alternative is given.